site/hero/pitController.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+include_once("hero.php");
+
+class PitController
+{
+  function oneOnOne($hero1, $hero2)
+  {
+    if ($hero1->CurrentHP <= 0) {
+		echo "Hero 1 has no HP!<br />";
+		return;
+	}
+    if ($hero2->CurrentHP <= 0) {
+		echo "Hero 2 has no HP!<br />";
+		return;
+	}
```

Comment:
you could setup https://github.com/markstory/lint-review to run against PR's. Robots telling you about bad tabs, spaces and new lines is less personal than a person :)